crates/apps_lib/src/client/rpc.rs

```diff
@@ -2446,3 +2466,68 @@ pub async fn compute_proposal_votes<C: Client + Sync>(
         delegator_voting_power,
     }
 }
+
+/// Query and display the IBC rate limit for the provided token
+pub async fn query_ibc_rate_limits(
+    context: &impl Namada,
+    args: args::QueryIbcRateLimit,
+) {
+    let token_alias = lookup_token_alias(context, &args.token, None).await;
+    let native_token = context.native_token();
+
+    match rpc::query_ibc_rate_limits(context.client(), &args.token).await {
+        Ok(IbcTokenRateLimits {
+            mint_limit,
+            throughput_per_epoch_limit,
+        }) => {
+            let (mint_str, throughput_str) = if args.token == native_token {
+                (
+                    mint_limit.to_string_native(),
+                    throughput_per_epoch_limit.to_string_native(),
+                )
+            } else {
+                (
+                    mint_limit.to_string(),
+                    throughput_per_epoch_limit.to_string(),
+                )
+            };
+            display_line!(
+                context.io(),
+                "IBC rate limits for token {}:\nGlobal mint limit: \
+                 {}\nThroughput limit: {} per epoch",
+                token_alias,
+                mint_str,
+                throughput_str
+            );
+        }
+        Err(_) => {
```

Comment:
Actually, given the way the ibc vp validates transactions, this logic is wrong. The sdk `query_ibc_rate_limits` functions returns an error if even one of the two keys is missing and in here we fallback to the default values for both, but the vp does actually support setting only one of the two and then using the default param for the other one. I'll try to rework this

Reply:
Done in 9a1b9b3

Reply:
thx, gotcha!